client/src/components/Header/Header.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react'
+import icon from '../../images/icon.svg'
+
+import './Header.scss'
+
+export interface HeaderProps {
+  title?: string
+  subtitle?: string
```

Comment:
Do we ever want to have a `Header` without a title/subtitle ? Why are these props optional ?

Reply:
Mmh nope indeed. Tbh I didn't know that the `?` was about optional prop haha.